Cargo.lock

```diff
@@ -1,3212 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
We want to keep the `Cargo.lock` file, as this is primarily a binary (not a library).